packages/phone-number-privacy/combiner/.env.celo-phone-number-privacy-stg

```diff
@@ -0,0 +1,27 @@
+MIN_INSTANCES=0
```

Comment:
TODO: should these be included here?

Reply:
Should no longer be relevant when using k8s